src/content/content.ts

```diff
@@ -13,7 +13,10 @@ const footerComponent = {
   nhsEngland: 'NHS England',
   supportLinks: 'Support links',
   links: {
-    accessibilityStatement: 'Accessibility statement',
+    accessibilityStatement: {
+      text: 'Accessibility statement',
+      url: 'https://notify.nhs.uk/accessibility',
```

Comment:
This should be just `/accessibility`, it will all sit behind the same host in a properly deployed environment.  The lower environments will (for now) reference the production CMS, however that should be configured in the gateway, not in the individual apps.